Cubical/Homotopy/Loopspace.agda

```diff
@@ -22,3 +24,38 @@ open import Cubical.Foundations.GroupoidLaws
 {- loop space map -}
 Ω→ : ∀ {ℓA ℓB} {A : Pointed ℓA} {B : Pointed ℓB} (f : A →∙ B) → (Ω A →∙ Ω B)
 Ω→ (f , f∙) = (λ p → (sym f∙ ∙ cong f p) ∙ f∙) , cong (λ q → q ∙ f∙) (sym (rUnit (sym f∙))) ∙ lCancel f∙
+
+
+module _ {ℓ : Level} {A : Type ℓ} {a b c : A} {p q : a ≡ b} {r s : b ≡ c} (α : p ≡ q) (β : r ≡ s) where
+  _◾ᵣ_ : (α : p ≡ q) (r : b ≡ c) → p ∙ r ≡ q ∙ r
+  (α ◾ᵣ r) i =
+     α i ∙ r
+
+  _◾ₗ_ : (p : a ≡ b) (β : r ≡ s) → p ∙ r ≡ p ∙ s
+  (p ◾ₗ β) i =
+    p ∙ β i
+
+  generalEH : (α ◾ᵣ r) ∙ (q ◾ₗ β) ≡ (p ◾ₗ β) ∙ (α ◾ᵣ s)
```

Comment:
Do you really need the ◾ᵣ and ◾ₗ ? The statement more or less the same if you unfold them